pages/handbook/product-engineering/playbooks/onboarding.mdx

```diff
@@ -0,0 +1,35 @@
+# Onboarding guideline of new Engineers
```

Comment:
Not much of a "guideline" than a laundry list of things I should have access to as a new joiner. 

Not bad, but as someone interested in joining, i would be interested in learning how my "first week" / "first month" at langfuse would be rather than just which systems I should have access to.